syne_tune/optimizer/schedulers/searchers/searcher.py

```diff
@@ -74,7 +77,7 @@ def on_trial_result(
         self,
         trial_id: int,
         config: Dict[str, Any],
-        metric: float,
+        metric: List[float],
```

Comment:
The naming should be metrics then.
```suggestion
        metrics: list[float],
```